src/__mocks__/context.ts

```diff
@@ -404,13 +402,14 @@ export class SyncIngestUpdateToPartInstanceContext extends RundownUserContext
 }
 
 // tslint:disable-next-line: max-classes-per-file
-export class ActionExecutionContext extends ShowStyleUserContext implements IActionExecutionContext {
+export class ActionExecutionContext extends ShowStyleUserContext implements ITV2ActionExecutionContext {
 	public currentPart: IBlueprintPartInstance
-	public currentPieceInstances: IBlueprintPieceInstance[]
+	public currentPieceInstances: Array<IBlueprintPieceInstance<PieceMetaData>>
```

Comment:
Why not use the `[]` syntax?
I would prefer that we used that syntax :)

Reply:
I would prefer them as well, but the current linting rules in this package enforce `Array<...<...>>` instead. We should start using the Sofie preset here - I think it allows `...<...>[]`.

Reply:
There are some things in the preset I'm not that big of a fan off. That being said I won't make too big of a fuss if we use it.
But maybe just start with chaning our linting rules to allow `...<...>[]`?